modules/odc-monitoring-your-project-metrics.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * applications/odc-monitoring-project-and-application-metrics-using-developer-perspective.adoc
+
+[id="odc-monitoring-your-project-metrics_{context}"]
+= Monitoring your project metrics
+
+After you create applications in your project and deploy them, you can use the *Developer* perspective in the web console to see the metrics for your project.
+On the left navigation panel of the *Developer* perspective, click *Monitoring* to see the *Dashboard*, *Metrics*, and *Events* for your project.
+
+.Procedure
+
+* Use the *Dashboard* tab to see graphs depicting the CPU, memory, and bandwidth consumption and network related information, such as the rate of transmitted and received packets and the rate of dropped packets.
++
+image::odc_project_dashboard.png[Monitoring dashboard]
++
+Use the following options to see further details:
+
+** Select the *Time Range* from the drop-down to determine the time frame for the data being captured.
```

Comment:
I believe per IBM Style Guide, each instance of "drop-down" throughout should be more specifically "drop-down list"?

Reply:
Using this instruction from the IBM style guide: "Refer to a drop-down list by its label, followed by list.' (Chpt 7- Computer interfaces, pg 209) I changed the mentions of drop-down to list in all cases that had a label. There is this one case which does not have a label on L-32 there I have changed it to drop-down list.